src/lib/stable/canister_methods/index.ts

```diff
@@ -0,0 +1,165 @@
+import { IDL } from '@dfinity/candid';
```

Comment:
These new functions need js docs, bonus points if you add js docs to all of the decorators